includes/utils/class-amp-wp-utils.php

```diff
@@ -58,4 +58,28 @@ protected static function _wp_translate_php_url_constant_to_key( $constant ) {
 
 		return false;
 	}
+
+	/**
+	 * Adds 'data-amp-layout' to the allowed <img> attributes for wp_kses() in a 'post' context.
+	 *
+	 * @since 0.7
+	 *
+	 * @param array  $context      Allowed tags and their allowed attributes.
+	 * @param string $context_type Type of context.
+	 * @return array $context      Filtered allowed tags and attributes.
+	 */
+	public static function add_layout( $context, $context_type ) {
+		if ( 'post' !== $context_type ) {
+			return $context;
+		}
+		$img            = isset( $context['img'] ) ? $context['img'] : array();
+		$context['img'] = array_merge(
+			$img,
+			array(
+				'data-amp-layout' => true,
+			)
+		);
+		return $context;
+	}
+
```

Comment:
This shouldn't be added to `AMP_WP_Utils` since it is marked for deletion in #876.

Reply:
Thanks, how about [AMP_HTML_Utils](https://github.com/Automattic/amp-wp/blob/develop/includes/utils/class-amp-html-utils.php)?

Reply:
Let's just put it in `AMP_Theme_Support` for now. It's the only class that is using it, and we'll probably break up that class into separate classes later anyway.

Reply:
Thanks, I'll move it to `AMP_Theme_Support`.

Reply:
This [commit](https://github.com/Automattic/amp-wp/pull/937/commits/011e47838e5154bb77af4f36010ff56d8bedd9eb) moves `add_layout()` to `AMP_Theme_Support`.